config/node/config.toml

```diff
@@ -150,6 +151,9 @@ CleanupLockedProofsInterval = "2m"
 GeneratingProofCleanupThreshold = "10m"
 UpgradeEtrogBatchNumber = 0
 BatchProofL1BlockConfirmations = 2
+SettlementBackend = "l1"
+AggLayerTxTimeout = "5m"
+AggLayerURL = ""
```

Comment:
are these needed if we don't use agg layer?

Reply:
let me test

Reply:
you can also check the default config and i think at least `AggLayerTxTimeout ` is not needed b/c it's the same as the defualt value https://github.com/0xPolygon/cdk-validium-node/blob/v0.6.4%2Bcdk/config/default.go#L179

Reply:
removed